lesson3/code-tasks/task-4.js

```diff
@@ -0,0 +1,27 @@
+const path = '../users.json';
+
+const loadUsers = (path) => {
+  return fetch(path)
+    .then(response => {
+      if (!response.ok) {
+        throw new Error('Network response was not ok.');
+      }
+
+      return response.json();
+    })
+    .then(users => {
+      let originalUsers = users
+      originalUsers.forEach(element => {
+        console.log(element.name);
+      });
+      console.log('================================SORTED===================================');
+      let sortedUsers = users.sort((a,b) => a.name.localeCompare(b.name))
```

Comment:
`sortedUsers` is not reassigned later, so you should use a `const` instead of `let` here